gradle.properties

```diff
@@ -8,15 +8,15 @@ description=Support for litematica on client to worldedit on server communicatio
 modVersion=1.0.0
 
 # Fabric
-minecraftVersion=1.20.1
-loaderVersion=0.14.21
-yarnMappings=1.20.1+build.10
+minecraftVersion=1.20.4
+loaderVersion=0.15.11
```

Comment:
Revert the loader version change. We don't need to bump this.

Reply:
Looking at https://github.com/FabricMC/fabric-loader/releases?page=2, it's not 100% clear when fabric loader started supporting 1.20.4, so, which is preferred?

- Keep at 0.14.21, which didn't support 1.20.2+
- Move to 0.14.23 which supported 1.20.3. IIRC 1.20.3 and 1.20.4 were mostly compatible with each other
- Move to 0.15.2 which is before 1.20.5 support

Reply:
0.14.23